.eleventy.js

```diff
@@ -149,6 +156,7 @@ module.exports = eleventyConfig => {
   eleventyConfig.addPairedShortcode('Aside', Aside);
   eleventyConfig.addPairedShortcode('Label', Label);
   eleventyConfig.addShortcode('LanguageList', LanguageList);
+  eleventyConfig.addNunjucksAsyncShortcode('Partial', Partial);
```

Comment:
Can you check if it is possible to use shortcodes inside a partial now? We had problems before to use a shortcode or an async shortcode inside another async shortcode.

Reply:
Partial is the only async shortcode currently though, right? Nested shortcodes just work fine, I did test that. But will add another test once #3783 lands 🙂